lib/passport/util.js

```diff
@@ -0,0 +1,29 @@
+const { exec } = require('child_process')
+
+const dependencies = {
```

Comment:
This could be moved to `package.json`, somewhere below the dependencies list. I think it would make it easier to find those additional dependencies.